phoenicis-dist/src/flatpak/org.phoenicis.playonlinux.desktop

```diff
@@ -7,6 +7,6 @@ Terminal=false
 Type=Application
 MimeType=application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;
 Icon=org.phoenicis.playonlinux
-Categories=Utility
+Categories=Games;Utility
```

Comment:
Should be "Game" (see https://standards.freedesktop.org/menu-spec/latest/apa.html).